### PR TITLE
CIVDT-1433: make cancel case CYA submit to update case service

### PR DESCRIPTION
**Pull request by tvaidotas**

### CODE REVIEW

**Review by jacemale-hmrc — APPROVED** [review]

File: test/controllers/cancelCase/CancellationReasonControllerSpec.scala [hunk 1]
```diff
@@ -21,11 +21,11 @@ import controllers.actions.FakeDataRetrievalAction
...
 import models.UserAnswers
-import models.requests.{DataRequest, IdentifierRequest, OptionalDataRequest}
+import models.requests._
 import pages.CheckModePage
 import pages.updateCase.UpdateAdditionalInformationPage
 import play.api.http.Status
-import play.api.mvc.{AnyContentAsEmpty, AnyContentAsFormUrlEncoded, Call, Result}
+import play.api.mvc._
 import play.api.test.FakeRequest
 import play.api.test.Helpers.{charset, contentType, defaultAwaitTimeout, redirectLocation, status}
```
Comment (jakebooker56) [inline test/controllers/cancelCase/CancellationReasonControllerSpec.scala:30]: imports

File: test/controllers/updateCase/DisclosureReferenceNumberControllerSpec.scala [hunk 2]
```diff
@@ -21,12 +21,12 @@ import controllers.actions.FakeDataRetrievalAction
...
-import models.requests.{DataRequest, IdentifierRequest, OptionalDataRequest}
+import models.requests._
 import org.scalamock.handlers.CallHandler
 import pages.CheckModePage
 import pages.updateCase.DisclosureReferenceNumberPage
 import play.api.http.Status
-import play.api.mvc.{AnyContentAsEmpty, AnyContentAsFormUrlEncoded, Call, Result}
+import play.api.mvc._
 import play.api.test.FakeRequest
 import play.api.test.Helpers.{charset, contentType, defaultAwaitTimeout, redirectLocation, status}
```
Comment (jakebooker56) [inline test/controllers/updateCase/DisclosureReferenceNumberControllerSpec.scala:31]: imports

File: test/controllers/updateCase/MoreDocumentationControllerSpec.scala [hunk 3]
```diff
@@ -21,11 +21,11 @@ import controllers.actions.FakeDataRetrievalAction
...
-import models.requests.{DataRequest, IdentifierRequest, OptionalDataRequest}
+import models.requests._
 import pages.CheckModePage
-import pages.updateCase.MoreDocumentationPage
+import pages.shared.MoreDocumentationPage
 import play.api.http.Status
-import play.api.mvc.{AnyContentAsEmpty, AnyContentAsFormUrlEncoded, Call, Result}
+import play.api.mvc._
 import play.api.test.FakeRequest
 import play.api.test.Helpers.{charset, contentType, defaultAwaitTimeout, redirectLocation, status}
```
Comment (jakebooker56) [inline test/controllers/updateCase/MoreDocumentationControllerSpec.scala:30]: imports

File: test/controllers/updateCase/UpdateAdditionalInformationControllerSpec.scala [hunk 4]
```diff
@@ -21,11 +21,12 @@ import controllers.actions.FakeDataRetrievalAction
...
+import models.requests._
 import pages.CheckModePage
-import pages.updateCase.{MoreDocumentationPage, UpdateAdditionalInformationPage}
+import pages.shared.MoreDocumentationPage
+import pages.updateCase.UpdateAdditionalInformationPage
 import play.api.http.Status
-import play.api.mvc.{AnyContentAsEmpty, AnyContentAsFormUrlEncoded, Call, Result}
+import play.api.mvc._
 import play.api.test.FakeRequest
 import play.api.test.Helpers.{charset, contentType, defaultAwaitTimeout, redirectLocation, status}
```
Comment (jakebooker56) [inline test/controllers/updateCase/UpdateAdditionalInformationControllerSpec.scala:31]: imports

File: test/services/updateCaseService/UpdateCaseServiceSpec.scala [hunk 5]
```diff
@@ -19,7 +19,7 @@ package services.updateCaseService
 import base.ServiceSpecBase
 import mocks.connectors.MockIvdSubmissionConnector
 import mocks.services.MockAuditService
-import models.audit.UpdateCaseAuditEvent
+import models.audit.{CancelCaseAuditEvent, UpdateCaseAuditEvent}
 import models.requests._
 import models.{UpdateCaseError, UpdateCaseResponse, UserAnswers}
```
Comment (jakebooker56) [inline test/services/updateCaseService/UpdateCaseServiceSpec.scala:24]: imports

**Review by jakebooker56 — APPROVED** [review]

**Pull request merged**